src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -79,18 +81,26 @@ class SettingsMenu extends Menu {
     onAudioTracks(model, audioTracks) {
         this.setupMenu(
             'audioTracks', 
+            this.localization.audioTracks,
             audioTracks, 
             (index) => this.api.setCurrentAudioTrack(index), 
             model.get('currentAudioTrack')
         );
     }
+    onAudioTrackIndex(model, trackIndex) {
```

Comment:
new functionality?

Reply:
I noticed while working on this the default audio track isn't selected on load. Added a listener for when it gets set to decorate the proper item. It now follows the same pattern as the rest of the menus. Created a ticket for it here, adding it to the PR description shortly: https://jwplayer.atlassian.net/secure/RapidBoard.jspa?rapidView=306&modal=detail&selectedIssue=JW8-10849